src/css/components/ItemsList.css

```diff
@@ -80,11 +80,20 @@
 .search-no-items {
   margin: 20px;
 }
-
+.item-details-icon {
+  order: 3;
+  border: none;
+  color: rgb(80, 79, 79);
```

Comment:
Gray with just the _slightest_ tinge of red? 😅

Reply:
we're so precise you can't tell 😂
.
.
.

actually we just select a color from the color picker and don't look at the numbers